packages/features/ee/workflows/lib/constants.ts

```diff
@@ -23,6 +23,7 @@ export const TIME_UNIT = [TimeUnit.DAY, TimeUnit.HOUR, TimeUnit.MINUTE] as const
 export const WORKFLOW_TEMPLATES = [
   WorkflowTemplates.CUSTOM,
   WorkflowTemplates.REMINDER,
+  WorkflowTemplates.RATING,
```

Comment:
TODO: havent figured out how to add this here now: 
<img width="301" alt="CleanShot 2024-03-26 at 13 58 09@2x" src="https://github.com/calcom/cal.com/assets/8019099/663a2190-cb16-4241-a2bf-2958f4a3d39a">